client/state/data-layer/wpcom/sites/stats/visits/index.js

```diff
@@ -20,6 +20,7 @@ export const fetch = ( action ) => {
 				query: {
 					unit: period,
 					date,
+					start_date,
```

Comment:
Similar to how `stat_fields` is handled, should we do `start_date: startDate` for consistency here?

Reply:
`startDate` is also used for the date selected. I wonder if we should either use `start_date` or `chartStart` to tell them apart 🤔

Reply:
`chartStart ` looks good to me!

Reply:
Addressed in [aae53f8](https://github.com/Automattic/wp-calypso/pull/96480/commits/aae53f88746e9325dfa9b10ada4603c24f8f37e6).